azure/durable_functions/entity.py

```diff
@@ -49,7 +49,7 @@ def handle(self, context: DurableEntityContext, batch: List[Dict[str, Any]]) ->
         for operation_data in batch:
             result: Any = None
             is_error: bool = False
-            start_time: datetime = datetime.now()
+            start_time: datetime = datetime.now(timezone.utc)
```

Comment:
Just to double check, the time in this PR is handled in the same way as the WebJobs extension right? I remember that you made some recent changes to this.

Reply:
Also yes, we eventually call `start_time.timestamp()`, which returns the number of seconds that have elapsed since January 1, 1970 UTC, and multiply it by 1000 to get the number of milliseconds, which is what the call on this field from the WebJobs side (`DateTimeOffset.FromUnixTimeMilliseconds`) expects

Reply:
Thanks for confirming!